src/integrationTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerEndpointTest.java

```diff
@@ -224,8 +230,21 @@ void dlq_case_event_message_should_be_stored_and_return_200_ok() throws Exceptio
         }
 
         @Test
-        void should_store_no_caseId_unprocessable_message_and_return_200_ok() throws Exception {
+        void dlq_case_event_message_should_not_be_stored_and_return_200_ok_when_feature_flag_disabled()
+                throws Exception {
 
+            when(launchDarklyFeatureFlagProvider.getBooleanValue(any(), any())).thenReturn(false);
+            String messageId = randomMessageId();
+
+            MvcResult result = postMessage(messageId, status().isCreated(), true);
+
+            String content = result.getResponse().getContentAsString();
+            assertEquals(201, result.getResponse().getStatus(), content);
```

Comment:
Are we missing an assertion here to ensure the message is not stored?

Reply:
Fixed (assertion added) in f4a2147